src/login/user-runtime-dir.c

```diff
@@ -216,10 +339,26 @@ static int run(int argc, char *argv[]) {
         if (r < 0)
                 return r;
 
-        if (streq(argv[1], "start"))
-                return do_mount(argv[2]);
+        if (streq(argv[1], "start")) {
```

Comment:
Not a matter, but it is time to convert this to `dispatch_verb()` ?

Reply:
i have some refactorings for user-runtime-dir in a git branch locally here. for example i wanted to rework it query its defaults from logind via varlink rather than dbus like it currently does it, I'd prefer to do that along with that